src/infrastructure/blockchain/ethereum/EthereumOracleAdapter.ts

```diff
@@ -20,26 +20,36 @@ class EthereumOracleAdapter implements OracleGateway {
   private emitter: EventEmitter;
   private guard: boolean;
   private contract: Contract;
+  private web3: Web3;
 
   constructor(web3: Web3, abi: AbiItem[], address: string) {
     // @ts-ignore
-    this.contract = new web3.eth.Contract(abi, address, { from: web3.eth.defaultAccount });
+    this.contract = new web3.eth.Contract(abi, address, {
+      from: web3.eth.defaultAccount,
+    });
 
     this.pendingResponses = [];
     this.emitter = new EventEmitter();
     this.guard = false;
+    this.web3 = web3;
 
     setInterval(() => this.sendPendingResponse(), ONE_SECOND_MILLIS);
   }
 
-  public async getRequests(fromBlock: number, toBlock: number): Promise<Request[]> {
-    const dataRequestedEvents = (await this.contract.getPastEvents('DataRequested', { fromBlock, toBlock }))
-        .map(event => _.pick(event.returnValues, ['id', 'url']))
-    ;
-
-    const delayedDataRequestedEvents = (await this.contract.getPastEvents('DelayedDataRequested', { fromBlock, toBlock }))
-        .map(event => _.pick(event.returnValues, ['id', 'url', 'validFrom']))
-    ;
+  public async getRequests(
+    fromBlock: number,
+    toBlock: number,
+  ): Promise<Request[]> {
+    const dataRequestedEvents = (
+      await this.contract.getPastEvents('DataRequested', {fromBlock, toBlock})
```

Comment:
Do you have JSLint enabled in your IDE? I think it would request to change `{fromBlock, toBlock}` to `{ fromBlock, toBlock }` . You don't have to do that, but just make sure you have JSLint enabled.